src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,53 @@
+package step2;
+
+public class Calculator {
+	private final Delimiter delimiter;
+
+	public Calculator(Delimiter delimiter) {
+		this.delimiter = delimiter;
+	}
+
+	public int calculate(String input) {
```

Comment:
```suggestion
		if (isBlank(input)) {
			return 0;
		}
```

parseOrZero을 추가하기보다 위와 같이 calculate() 내에서 처리하는 것은 어떨까?